devices/Paexo/test-application/paexo-test-application.cpp

```diff
@@ -23,44 +29,94 @@ int main() {
     std::string inPort = "/Paexo/WearableActuatorsCommand/input:i";
     std::string outPort = "/Paexo/WearableActuatorsCommand/output:o";
 
-    if (!yarp::os::Network::exists(inPort))
-    {
-        yError() << "Port " << inPort << " does not exists";
-        return -1;
-    }
+    std::vector<std::string> PaexoActuators = {"Paexo::motor::LeftMotor", "Paexo::motor::RightMotor"};
 
-    BufferedPort<wearable::msg::WearableActuatorCommand> port;
-    if(!port.open(outPort))
-    {
-        yError() << "Failed to open port " << outPort;
-        return -1;
-    }
+    std::vector<std::string> PaexoActuatorPortNames = {"/Paexo/motor/LeftMotor"};
+    std::vector<std::unique_ptr<YarpBufferedPort>> PaexoActuatorPorts;
+
+//    if (!yarp::os::Network::exists(inPort))
+//    {
+//        yError() << "Port " << inPort << " does not exists";
+//        return -1;
+//    }
```

Comment:
We could remove those lines

Reply:
Done in https://github.com/robotology/wearables/pull/124/commits/faefcae8f616d0b1a912f78e50a7340c42e037a0